src/csharp/Microsoft.Spark/ML/Feature/Bucketizer.cs

```diff
@@ -243,5 +254,17 @@ public Bucketizer SetHandleInvalid(string value)
         {
             return WrapAsBucketizer(_jvmObject.Invoke("setHandleInvalid", value.ToString()));
         }
+        
+        /// <summary>
+        /// The reference we get back from each call isn't usable unless we wrap it in a new dotnet
+        /// <see cref="Bucketizer"/>
+        /// </summary>
+        /// <param name="obj">The <see cref="JvmObjectReference"/> to convert into a dotnet
+        /// <see cref="Bucketizer"/></param>
+        /// <returns><see cref="Bucketizer"/></returns>
```

Comment:
missing comment.

Reply:
or let's remove this comment block since it's a private function:
```C#
private static Bucketizer WrapAsBucketizer(object obj) => new Bucketizer((JvmObjectReference)obj);
```
(Check `DataFrameNaFunctions.cs` for example)